.circleci/config.yml

```diff
@@ -628,7 +629,9 @@ jobs:
             make install
             make generate_api_code
             mix deps.get
-            mix test
+            echo $(circleci tests split ./tags.txt)
```

Comment:
I think that with a little bit of `grep` , `awk` and `tr` you could go without this file?

grep -r "@moduletag" | awk '{print $3}' | tr -d ':'